.github/workflows/ci.yml

```diff
@@ -109,8 +109,8 @@ jobs:
         #
         # ...to:
         #
-        #   toolchain: $ {{ ./cargo.sh --version matrix.toolchain }} # hypothetical syntax
-        ZC_TOOLCHAIN="$(./cargo.sh --version ${{ matrix.toolchain }})"
+        #   toolchain: $ {{ ./cargos.sh --version matrix.toolchain }} # hypothetical syntax
```

Comment:
Is there a reason not to leave this named `cargo.sh`? (I'll be honest I'm not sure what the etymology of `cargos` is as opposed to `cargo`)

Reply:
It is an unfortunately mundane one. I originally kept it named `cargo.sh` and made `cargo.bat` to match. As it turns out though, when `cargos` runs a `cargo` command, Windows will run `cargo.bat` and not the installed `cargo` binary. So it effectively causes `cargos` to recursively call itself when it intends to invoke `cargo`. The easy fix was to use a different name, and I figured if I had to call it `cargos` on Windows then I should probably just make it uniform.

Reply:
Ah, that makes sense. My preference would be to keep `cargo.sh` named as-is, and rename the `.bat` file. Maybe `win-cargo.bat`? I'll leave it up to you what you want to name it.